gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -272,15 +272,8 @@ def self._handle_missing_method_signature(receiver, original_method, callee)
     receiving_method = receiving_class.instance_method(callee)
     if receiving_method != original_method && receiving_method.original_name == original_method.name
       aliasing_mod = receiving_method.owner
-
-      # Note, this logic is duplicated above, make sure to keep changes in sync.
-      if method_sig.check_level == :always || (method_sig.check_level == :tests && T::Private::RuntimeLevels.check_tests?)
```

Comment:
Just checking: since this was replaced with a call to `unwrap_method`, is the check_level check done in there instead?

Reply:
Yeah, that's right.

Reply:
Done in `wrap_method_if_needed`: https://github.com/sorbet/sorbet/blob/1275104e2c5ebd9eb799df08a56f089493304237/gems/sorbet-runtime/lib/types/private/methods/call_validation.rb#L37-L39